hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRequest.java

```diff
@@ -174,8 +174,8 @@ protected KeyArgs resolveBucketAndCheckOpenKeyAcls(KeyArgs keyArgs,
     return resolvedArgs;
   }
 
-  protected void validateKeyName(OzoneManager ozoneManager, String keyName)
-      throws OMException {
+  protected void validateKeyName(OzoneManager ozoneManager, KeyArgs keyArgs,
+      String keyName) throws OMException {
```

Comment:
If you are passing the keyArgs, there is no need to pass keyName. You can get is from keyArgs here.